python/ray/serve/tests/test_metrics.py

```diff
@@ -56,9 +56,9 @@ def verify_metrics(do_assert=False):
             "serve_deployment_request_counter",
             "serve_deployment_replica_starts",
             # histogram
-            "deployment_processing_latency_ms_bucket",
-            "deployment_processing_latency_ms_count",
-            "deployment_processing_latency_ms_sum",
```

Comment:
Do you know why this test was passing before if these metrics didn't exist yet?

Reply:
Sorry I misunderstood, this is a different metric. Still want to know if it was passing or not running though

Reply:
It is same metric, the reason it can pass before because we check whether the `deployment_processing_latency_ms_sum` string is inside the metrics blob. 
For this change, I just want to make the test check more restrict.

Reply:
Ah makes sense!